packages/utils/package-manager.ts

```diff
@@ -13,17 +13,17 @@ import * as path from "path";
  */
 
 export function getPackageManager(): string {
-	const hasLocalNPM = fs.existsSync(path.resolve(process.cwd(), "package-lock.json"));
-	const hasLocalYarn = fs.existsSync(path.resolve(process.cwd(), "yarn.lock"));
-	if (hasLocalNPM) {
-		return "npm";
-	} else if (hasLocalYarn) {
-		return "yarn";
-	} else if (spawn.sync("yarn", [" --version"], { stdio: "ignore" }).error) {
-		return "npm";
-	} else {
-		return "npm";
-	}
+    const pwd = path.resolve(process.cwd());
+    const hasLocalNPM = fs.existsSync(path.resolve(pwd, '/package-lock.json'));
+    const hasLocalYarn = fs.existsSync(path.resolve(pwd + '/yarn.lock'));
```

Comment:
Let's use `execa` to spawn a process. Also, let's not check the lock file because a first time installation would fail.

Use execa to run a synchronous command: yarn -v

If the result is an error, it means the user doesn't have yarn installed